ckanext/datajson/tests/test_export.py

```diff
@@ -1,26 +1,26 @@
 from __future__ import print_function
+
 from future import standard_library
+
 standard_library.install_aliases()
 import json
-import six
 import zipfile
 
 import ckanext.harvest.model as harvest_model
-
+import six
 from ckan.tests import factories
-
 from ckan.tests.helpers import reset_db
 
-
 if six.PY2:
     from ckan.tests.helpers import FunctionalTestBase
     inherit = FunctionalTestBase
 else:
     inherit = object
-    from flask import Response

```

Comment:
Why do we remove this?

Reply:
I don't believe it was removed. VSCode rearranged the imports.